### PR TITLE
Fallback ('Did not fetch typename')

### PR DESCRIPTION
**Pull request by JuliusKarl**
Error handling fallback when content is 

- Missing
- Inaccessible
- Archived
- Unpublished

When encountered, the user is redirected to the UoA standard error page _this.router.navigate([/error/500])_. Also the graphql queries are updated so that it works correctly again, this pr is more of a quick hotfix rather than a full feature.

### CODE REVIEW

**Comment by JuliusKarl** [comment]
Changes

- Including 'National' Font
- Missing content fallback
- Updating  GraphQL Queries

Note

- Some things might look a bit ugly at the moment, I have full-feature branches that fixes those issues this branch just fixes the critical issues
- You won't be able to access SubHubs without SSO, this is fixed in my `subhub-page` branch later.

File: research-hub-web/package.json [hunk 1]
```diff
@@ -94,8 +95,9 @@
     "fibers": "^5.0.0",
     "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~5.1.1",
+    "karma": "^5.1.1",
     "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage": "^2.0.3",
     "karma-coverage-istanbul-reporter": "~3.0.2",
```
Comment (rosemcc) [inline research-hub-web/package.json:101]: uninstall the old coverage reporter if not required

File: research-hub-web/src/app/components/equipment/equipment.component.ts [hunk 2]
```diff
@@ -43,20 +44,11 @@ export class EquipmentComponent implements OnInit {
...
-
-  /**
-   * Function that loads the equipment/collection depending on if a slug is present.
-   */
-  private async _loadContent() {
     /**
-     * If this.slug is defined, we're loading an individual equipment,
-     * therefore run the corresponding query. If not, return all equipment.
+     * If this.slug is defined, we're loading an individual article,
+     * therefore run the corresponding query. If not, return all articles.
```
Comment (rosemcc) [inline research-hub-web/src/app/components/equipment/equipment.component.ts:51]: minor fix for the comments - article should be equipment

File: research-hub-web/src/app/components/search-results/subhubs.component.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit, Type } from '@angular/core';
```
Comment (rosemcc) [inline research-hub-web/src/app/components/search-results/subhubs.component.ts:1]: is this supposed to be under search-results? is it just temporary?

Comment (JuliusKarl) [reply research-hub-web/src/app/components/search-results/subhubs.component.ts]: oh yeah i think was a drag and drop mistake

**Review by rosemcc — CHANGES_REQUESTED** [review]
All good just a couple of minor comments to address

**Review by rosemcc — APPROVED** [review]

**Pull request merged**